src/functional.test.ts

```diff
@@ -1,6 +1,8 @@
 import { withMonorepoProjectEnvironment } from '../tests/functional/helpers/with';
 import { buildChangelog } from '../tests/functional/helpers/utils';
 
+jest.setTimeout(10_000);
+
 describe('create-release-branch (functional)', () => {
```

Comment:
It would be nice if we could write a functional test involving nested workspaces, but we can do that later.